products/samsungmobile.md

```diff
@@ -157,7 +387,7 @@ releases:
     releaseDate: 2021-03-03
 
 -   releaseCycle: "Galaxy A32"
-    support: true
+    support: 2022-10-01 # Approximate from here: https://9to5google.com/2022/10/04/samsung-android-12-update-rollout/
```

Comment:
Upgradable to Android 13 according to https://www.gsmarena.com/samsung_galaxy_a32-10753.php. Should be marked supported.